docs/metrics.md

```diff
@@ -28,20 +29,39 @@ This is the metrics collection plan for Lockbox's alpha release. It is more of a
 
 Best viewed in something that can render Markdown.
 
-## TL;DR Metrics Priorities for Alpha
+## TL;DR Metrics Implemented for Alpha Thus Far
+
+These are the metrics that have been implemented in the pull request that included the edits to this doc.
+
+All events are under the **category: lockbox**. The `extra` field always contains `fxaid` where possible (i.e. after FxA auth). In the future also the `itemid` where relevant.
+
+1. `startup` fires when the webextension is loaded. **method**:click **objects**: addon, webextension.
+
+2. `iconClick` fires when someone clicks the toolbar icon. **method**:startup **objects**: toolbar.
+
+3. `render` fires when one of the views are rengered. **method**:render **objects**: firstrun, manage, popupUnlock
+
+4. `signIn` fires when someone clicks the signin button during firstrun. **method**: render **objects**: signInPage.
```

Comment:
This seems rather confusing to me; based on the name, I'd expect this to fire when the first page of the wizard is rendered, not when we pop up the FxA page. Maybe call this something like `fxaSignIn`?